app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,46 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+#!/bin/sh
+<%
+    curl_headers = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
```

Comment:
You're repeating `-H` a lot. Can't you do that in the curl part of the template?
```suggestion
    headers = ['Content-Type: application/json', 'Accept: application/json', "Authorization: Bearer #{@auth_token}"]
```

Other than the readability, it also decouples from Curl to just a set of HTTP parameters. This may help if another client should be used.